charts/studio/templates/deployment-studio-backend.yaml

```diff
@@ -13,10 +13,11 @@ spec:
       {{- include "studio-backend.selectorLabels" . | nindent 6 }}
   template:
     metadata:
-      {{- with .Values.studioBackend.podAnnotations }}
       annotations:
+      {{- with .Values.studioBackend.podAnnotations }}
         {{- toYaml . | nindent 8 }}
       {{- end }}
+        checksum/studio-config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
```

Comment:
What does this do?

Reply:
It triggers restart of the pod when ConfMap is changed

https://helm.sh/docs/howto/charts_tips_and_tricks/#automatically-roll-deployments

Reply:
:+1: